src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep3.java

```diff
@@ -0,0 +1,212 @@
+package org.sagebionetworks.web.client.widget.accessrequirements.createaccessrequirement;
+
+import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
+
+import java.util.HashSet;
+
+import org.sagebionetworks.repo.model.AccessControlList;
+import org.sagebionetworks.repo.model.ManagedACTAccessRequirement;
+import org.sagebionetworks.repo.model.ResourceAccess;
+import org.sagebionetworks.repo.model.principal.TypeFilter;
+import org.sagebionetworks.web.client.DisplayConstants;
+import org.sagebionetworks.web.client.SynapseJavascriptClient;
+import org.sagebionetworks.web.client.utils.CallbackP;
+import org.sagebionetworks.web.client.widget.search.SynapseSuggestBox;
+import org.sagebionetworks.web.client.widget.search.UserGroupSuggestion;
+import org.sagebionetworks.web.client.widget.search.UserGroupSuggestionProvider;
+import org.sagebionetworks.web.client.widget.table.modal.wizard.ModalPage;
+import org.sagebionetworks.web.client.widget.team.UserTeamBadge;
+import org.sagebionetworks.web.shared.exceptions.NotFoundException;
+import org.sagebionetworks.web.shared.users.AclUtils;
+import org.sagebionetworks.web.shared.users.PermissionLevel;
+
+import com.google.common.util.concurrent.FluentFuture;
+import com.google.common.util.concurrent.FutureCallback;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+/**
+ * Second page of creating an access requirement (ACT)
+ * 
+ * @author Jay
+ *
+ */
+public class CreateManagedACTAccessRequirementStep3 implements ModalPage, CreateManagedACTAccessRequirementStep3View.Presenter {
+	CreateManagedACTAccessRequirementStep3View view;
+	ModalPresenter modalPresenter;
+	ManagedACTAccessRequirement accessRequirement;
+	SynapseJavascriptClient jsClient;
+	AccessControlList originalAcl;
+	String reviewerPrincipalId;
+	UserTeamBadge reviewerUserTeamBadge;
+	public static final String TOO_MANY_ACL_ENTRIES_ERROR = "This Access Requirment has an ACL that contains more than one item.  Beware, this dialog will destroy other entries on save.";
```

Comment:
I thought we were OK with having >1 ACL entry. Did this change?

Reply:
Adam indicated that there should be "only 1 non-ACT reviewer", so that's what the UI supports (knowing that the backend supports an Access Control **_List_**)